docs/development/translations.rst

```diff
@@ -0,0 +1,300 @@
+
+.. getting_help_translations:
+
+Getting help about translations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Should you need help, you can do one of the following:
+
+* Post a message in the `translation category of the SecureDrop forum`_
+* Chat in the `SecureDrop instant messaging channel`_
+* Read the `Weblate documentation`_
+
+How-to Guides
+-------------
+
+.. _how_to_register_on_weblate_email:
+
+How to register an account on Weblate using an email address
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+#. Visit the `Weblate registration page`_.
+#. Fill the form **Register using email** and click **Register**.
+#. Check your email for a message from **weblate@securedrop.org** with the subject **[Weblate] Your registration on Weblate**.
+#. That message contains a confirmation link. Click that link to complete your registration.
+
+.. _how_to_register_on_weblate_github:
+
+How to register an account on Weblate using a GitHub account
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+#. Visit the `Weblate registration page`_.
+#. Click on the GitHub icon, under **Third party registration**.
+#. Log into GitHub unless you already are.
+#. Click the green **Authorize freedomofpress** button.
+
+The authorization request looks like this:
+
+|GitHub authorization page screenshot|
+
+.. _how_to_manage_your_preferred_languages_weblate:
+
+How to manage your preferred languages on Weblate
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+#. Visit the `Weblate dashboard`_.
+#. Click the **Manage your languages** button.
+#. Select the languages your want to translate.
+#. Click the **Save** button.
+
+|Weblate manage languages screenshot|
+
+.. _how_to_translate_a_language_weblate:
+
+How to translate a language on Weblate
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+#. Visit the `Weblate dashboard`_.
+#. Click on the **component** in order to display the list of languages in which it is translated.
+#. Click the **Translate** button.
+#. :ref:`Start translating <how_to_translate_a_phrase_weblate>`.
+
+|Weblate translations screenshot|
+
+.. _how_to_translate_a_phrase_weblate:
+
+How to translate a phrase on Weblate
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+#. :ref:`Select a language <how_to_translate_a_language_weblate>`.
+#. Read the translatable string in the text area labelled **Source**.
+#. Review the suggested translations if there are any in the **Glossary** sidebar.
+#. Review the contextual information about the source string in the **Source information** sidebar, like its location in our source code.
+#. If a screenshot of the SecureDrop user interface is vailable, read the source string in context.
+#. Input your translation in the **Translation** test area near the source string.
+#. Click **Save**. The next untranslated string will apear automatically.
+
+|Weblate translate screenshot|
+
+
+
+
+
+Reference
+---------
+
+
+
+Background information
+----------------------
```

Comment:
This and following sections should be a level below "Reference", to avoid having an emtpy "Reference" section in the outline view.

Reply:
My intention was to create four "level 2" sections:

- **Getting started** (not exactly the same as Tutorials, I think of those as introduction to the other 4 types defined in the Divio model)
- (**Tutorials** but there aren't any at the moment)
- **How-to Guides**
- **Reference**
- **Background information**

I totally agree there is no point in leaving empty sections. I had the intention of putting in there some glossaries-related content that I left out when I decided that this was enough to demo what things could look like. And I left **References** as a placeholder... which in hindsight wasn't obvious. Next time I'll leave a comment 🙂

Reply:
> Getting started (not exactly the same as Tutorials, I think of **those** as introduction to the other 4 types defined in the Divio model)

Oh, by the way, I was thinking that both the "Getting started" for translators and "Getting started" for developers could live here, and both the i18n and l10n could indeed be consolidated into one. I can picture that with adequate headers it wouldn't make things unambiguous or confusing, but I guess that's something that would need giving it a try to validate.

Reply:
With the glossaries in place in the **Reference** section this should be clearer. I added _TODO_ notes here and there to signify missing content.

Reply:
I understand the design intention a bit better now. I think the navigational outline improves the original in many ways, but there are a few things that I think are still challenging for a reader:

1) The distinction between "Background", "Reference" and "How-to" is not at all obvious to a reader attempting to locate an answer quickly. Especially the "Background" and "How-to" sections contain some very similar instructional information, and are both richly populated.<br><br>My bias would be to move as much into the "How-to" section as possible (possibly giving "How-to" a bit more hierarchical structure) and to keep "Background" very short. 

2) If our main goal with the "Reference" section is to provide a glossary for translators, I would just call it "Glossary" and keep it super-simple: word/definition. Leave everything else for the other sections. Right now it feels a bit unclear in its scope.

3) I would put the "Reference" or "Glossary" section at the end. So 1) Quick start, 2) Background, 3) How to ..., 4) Glossary.

Reply:
I'll re-read this before making changes, but it makes sense to me.

> My bias would be to move as much into the "How-to" section as possible (possibly giving "How-to" a bit more hierarchical structure) and to keep "Background" very short.

At first sight, this is the part that requires most work. I don't visualize it clearly yet but I get the idea in principle and it makes sense to me. Once I refresh the context in my mind I'll have a go at moving the most straightforward parts first.

Reply:
Done.